.github/workflows/go.yml

```diff
@@ -21,6 +21,37 @@ env:
   GOPROXY: https://proxy.golang.org
 
 jobs:
+  short-test:
+    name: Short test
```

Comment:
Even better if: the name is something like `Short test (only checks XYZ)`

Reply:
I'm pretty sure that name would not fit GitHub UI :)